qdrant-landing/content/documentation/guides/common-errors.md

```diff
@@ -32,6 +32,69 @@ ulimit -n 10000
 
 Please note, the command should be executed before you run Qdrant server.
 
+## Incompatible file system
+
+Qdrant have a [set of requirements](https://qdrant.tech/documentation/guides/installation/#storage) for persistent file storage.
+The most important requirement is that file system **must** be [POSIX-compatible](https://www.quobyte.com/storage-explained/posix-filesystem/).
+
+
+Starting from v1.15.0 Qdrant performs runtime check of file system compatibility on start.
+If it detects an unknown file system, you can see a warning like this:
+
+```text
+WARN qdrant: There is a potential issue with 
+the filesystem for storage path ./storage. Details:
+HFS/HFS+ filesystem support is untested
+```
+
+If runtime check fails, you might see an error message:
+
+```text
+ERROR qdrant: Filesystem check failed for storage path ./storage.
+Details: FUSE filesystems may cause data corruption due to caching issues
+```
+
+If an error like this is reported, it is NOT safe to continue working with current configuration and you're at risk of losing your data.
+
+Most common errors you might see, if you continue using Qdrant with incompatible file system:
+
+```text
+ERROR
+Panic occurred in file /qdrant/lib/gridstore/src/gridstore.rs at line 53:
+called `Result::unwrap()` on an `Err` value: OutputTooSmall { expected: 4, actual: 0 }
+```
+
+or
+
+```text
+ERROR
+Service internal error: task XXX panicked with message
+"called `Result::unwrap()` on an `Err` value: OutputTooSmall { expected: 4, actual: 0 }"
+```
+
+It might be also possible that vector data will be lost (set to all zeros) after service restart.
+
+
+### How to avoid Incompatible file system?
+
+Most common used configuration of incompatible file system is usage of WSL-baced Docker containers in Windows.
+When you mount Windows folder into Qdrant docker container, the Windows hyper visor creates a shared mount, which is not fully POSIX-compatible.
```

Comment:
AFAIK this isn't FUSE, because it's mounted by the kernel directly? FUSE is for user space mounts, not for special mounts like this.

Or do you see the 'FUSE' error message in that specific scenario?

In any case, I changed the text. If you don't agree we can change it back 👍